src/schrodinger/sketcher/menu/cut_copy_action_manager.cpp

```diff
@@ -76,6 +76,13 @@ void CutCopyActionManager::initCopyAsMenu()
                 QString::fromStdString(label), this, slot);
             action->setData(QVariant(is_reaction_format));
         }
+        // Add a separator and the option to export as an image
+        auto separator = m_copy_as_menu->addSeparator();
+        separator->setData(QVariant(is_reaction_format));
+        auto action = m_copy_as_menu->addAction("Image", this, [this]() {
+            emit copyAsImageRequested(getSubset());
+        });
+        action->setData(QVariant(is_reaction_format));
     };
 
```

Comment:
This is only tangentially related to your change, but it might be helpful to have a comment here mentioning that the action's data is used to control visibility in the updateActions method.  At first, I was confused as to why you were setting data on the separator.

Reply:
added a comment